blant.c

```diff
@@ -2518,29 +2523,37 @@ int RunBlantFromEdgeList(int k, int numSamples, int numNodes, int numEdges, int
     return RunBlantInThreads(k, numSamples, G);
 }
 
+
+
+
+#if GEN_SYN_GRAPH
+
 //
 // Generate Synthetic Graph by Stamping graphlet from concentration distribution
 //
 
+void reset_global_maps(int k_new) {_k = k_new; SetGlobalCanonMaps(); LoadMagicTable();}
+
 //Some Stats functions for later usage
 double* convertPDFtoCDF(double pdf[], double cdf[])
 {
     int i;
     cdf[0] = pdf[0];
     for(i=1; i<_numConnectedCanon; i++) cdf[i] = cdf[i-1] + pdf[i] < 1 ? cdf[i-1] + pdf[i] : 1;
-    if((1-cdf[i-1]) < 0.0001) cdf[i-1] = 1;
+    if((1-cdf[i-1]) < 0.0005) cdf[i-1] = 1;
     if(cdf[i-1] != 1) 
         printf("Increase rounding error margin. Last term of CDF is: %lf\n", cdf[i-1]);
     assert(cdf[i-1] == 1);
     return cdf;
 }
 
-double* copyConcentration(double src[], double dest[]){int i;for(i=0;i<_numConnectedCanon;i++)dest[i]=src[i];return dest;}
+double* copyConcentration(double src[], double dest[]) {for(int i=0; i<_numConnectedCanon; i++) dest[i] = src[i]; return dest;}
```

Comment:
Please leave variable declarations outside the for loop; not all compilers (eg MacOS) accept them inside for loops, which is why I moved them out.